packages/dashboard/src/DashboardEvents.ts

```diff
@@ -2,22 +2,30 @@ import type { EventHub } from '@deephaven/golden-layout';
 
 export const CREATE_DASHBOARD = 'CREATE_DASHBOARD';
 
-export interface CreateDashboardPayload {
+export interface CreateDashboardPayload<T = unknown> {
   pluginId: string;
   title: string;
-  data: unknown;
+  data: T;
```

Comment:
Why the generic? The data comes from the plugin and we only really care it's serializable

Reply:
So the listen/emit events can add typing for payload. For the emit function which will come from the plugin, can add typing so we know we're not passing a malformed payload.